doc/gemma_online.md

```diff
@@ -0,0 +1,34 @@
+
+Naast gebruik van user stories worden standaarden ook agile ontwikkeld. In plaats van een langdurig ontwikkelproces wordt een onderdeel van een standaard in een korte cyclus uitgewerkt en afgestemd met de gebruikers. 
+
+== Transparantie
+De API standaarden worden volledig transparant en openbaar ontwikkeld. Alle user stories, documentatie, broncode, ontwerpen en andere zaken worden in een openbare Github repository vastgelegd. Daar kunnen belanghebbenden ook zelf userstories en bevindingen doorgeven die door het ontwikkelteam besproken, beoordeeld en opgepakt kunnen worden. 
+
+
+== Zaakgericht Werken
+De eerste standaard die op deze manier wordt ontwikkeld is een groep standaarden ter ondersteuning van het <a href="https://www.gemmaonline.nl/index.php/Thema_Zaakgericht_werken">Zaakgericht Werken</a>. De opvolger van de <a href="https://www.gemmaonline.nl/index.php/Zaak-_en_Documentservices">Zaak- Documentservices</a>, een op <a href="https://www.gemmaonline.nl/index.php/Sectormodel_Zaken:_StUF-ZKN">StUF ZKN</a> gebaseerde koppelvlakstandaard, is opgezet volgens de <a href" https://www.gemmaonline.nl/index.php/GEMMA_Architectuur">GEMMA 2</a> referentie architectuur. 
+
+De Github repository waar deze standaard in ontwikkeld wordt staat hier: <a href="https://github.com/vng-realisatie/gemma-zaken">https://github.com/vng-realisatie/gemma-zaken</a>. De Github repository is dus een ontwikkelomgeving, in software termen zou dit zelfs een nightly build genoemd kunnen worden. De versie van de standaard in ontwikkeling zoals die getoond wordt tijdens een sprint demo staat op een aparte omgeving welke te vinden is op <a href="https://ref.tst.vng.cloud/">https://ref.tst.vng.cloud/</a> en is te vergelijken met een stable build.
+
+De volgende referentie componenten zijn (vooralsnog) onderdeel van de Zaakgericht Werken standaard:
+<ul>
+<li><a href="https://ref.tst.vng.cloud/standaard/apis/zrc">Zaakregistratie component</a></li>
+<li><a href="https://ref.tst.vng.cloud/standaard/apis/drc">Documentregistratie component</a></li>
+<li><a href="https://ref.tst.vng.cloud/standaard/apis/ztc">Zaaktypecatalogus</a></li>
+<li><a href="https://ref.tst.vng.cloud/standaard/apis/brc">Besluitregistratie component</a> (nieuw geidentificeerd en toegevoegd op basis van de user stories)</li>
+</ul>
+
+Daarnaast is als <a href="https://nl.wikipedia.org/wiki/Mock-up">mockup</a> het <a href="https://ref.tst.vng.cloud/standaard/apis/overige">Overigeregistratiecomponent (ORC)</a> gemaakt, bedoeld voor die gegevens die wel in de user story thuis horen maar niet in het RGBZ opgenomen zijn. Hiermee is het mogelijk om de andere registratiecomponenten met bijbehorende API's die nog niet gerealiseerd zijn te simuleren en zo een user story te kunnen realiseren.
```

Comment:
De status, inhoud, eisen en functionele inzet van het ORC is nog een beetje vaag :( Niet te veel over zeggen.

Reply:
Dat vond ik ook een lastige maar ik wilde er wel iets over zeggen omdat het ding wel op de ref.tst omgeving staat. Er is ook geen GEMMA beschrijving van dus inderdaad een beetje vaag allemaal.